src/cleanup.cpp

```diff
@@ -54,6 +54,7 @@ void
 clean_up_dir(const std::string& subdir,
              uint64_t max_size,
              uint32_t max_files,
+             time_t max_old,
```

Comment:
```suggestion
             time_t max_age,
```

And similar elsewhere.

Reply:
done